dxr/static/templates/path_line.html

```diff
@@ -1,4 +1,10 @@
 {%- if not is_first_or_only -%}
 <span class="path-separator">/</span>
 {%- endif -%}
-<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>{{ display_path }}</a>
\ No newline at end of file
+{%- if not is_binary and not is_dir -%}
+<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
+{%- endif -%}
+    {{ display_path }}
+{%- if not is_binary -%}
```

Comment:
Since this condition isn't the same as the one enclosing the `<a>`, are there any situations in which we might end up printing the close tag without the opener?